test/components/views/beacon/BeaconViewDialog-test.tsx

```diff
@@ -84,13 +80,11 @@ describe("<BeaconViewDialog />", () => {
         matrixClient: mockClient as MatrixClient,
     };
 
-    const getComponent = (props = {}) => mount(<BeaconViewDialog {...defaultProps} {...props} />);
+    const getComponent = (props = {}): RenderResult => render(<BeaconViewDialog {...defaultProps} {...props} />);
 
-    const openSidebar = (component: ReactWrapper) =>
-        act(() => {
-            findByTestId(component, "beacon-view-dialog-open-sidebar").at(0).simulate("click");
-            component.setProps({});
-        });
+    const openSidebar = (getByTestId: RenderResult["getByTestId"]) => {
+        fireEvent.click(getByTestId("beacon-view-dialog-open-sidebar"));
+    };
```

Comment:
If we were to import `screen` from `@testing-library/react` then we would be able to call this function without an argument, as we could fire the click event on `screen.getByTestId(...)`